src/Raven.Server/Documents/PeriodicBackup/Restore/RestoreSnapshotTask.cs

```diff
@@ -92,9 +92,9 @@ protected override async Task InitializeAsync()
             FilesToRestore.RemoveAt(0);
         }
 
-        protected override void OnAfterRestore()
+        protected override async Task OnAfterRestoreAsync()
         {
-            base.OnAfterRestore();
+            await base.OnAfterRestoreAsync();
```

Comment:
Is there common code of the impl of `AbstractRestoreBackupTask` at `OnAfterRestoreAsync`
If no, it is better to make `OnAfterRestoreAsync` abstract
(I don't see any other call to `base.OnAfterRestoreAsync`)

Reply:
yes, there's common code of `OnAfterRestoreAsync` at `AbstractRestoreBackupTask`, used by non-sharded backup restore and by snapshot restore